ph5/clients/ph5tostationxml.py

```diff
@@ -630,8 +630,11 @@ def read_stations(self):
                                                                )
                         obs_station.selected_number_of_channels = 0
                     if obs_station not in all_stations:
-                        all_stations.append(obs_station)
-        return all_stations
+                        all_stations.add(obs_station)
+        all_stations_list = list(all_stations)
+        all_stations_list.sort()

```

Comment:
Does sorting this list objects in this manor work as expected? Sorting is also a very expensive operation. Ideally this method should return an iterable like yield so that the order is implicit. I think for now we can do this but eventually we should probably remove the sort.

Reply:
Yeah we should probably remove the sort eventually